scene/gui/color_picker.cpp

```diff
@@ -89,6 +89,8 @@ void ColorPicker::_notification(int p_what) {
 			shape_popup->set_item_icon(shape_popup->get_item_index(SHAPE_VHS_CIRCLE), theme_cache.shape_circle);
 			shape_popup->set_item_icon(shape_popup->get_item_index(SHAPE_OKHSL_CIRCLE), theme_cache.shape_circle);
 
+			btn_shape->set_icon(shape_popup->get_item_icon(current_shape));
```

Comment:
```suggestion
			if (current_shape != SHAPE_NONE) {
				btn_shape->set_icon(shape_popup->get_item_icon(current_shape));
			}
```

There is this case when the shape is hidden completely, then this won't work. So we need to guard against it.

Also, could you please amend the commit message. The current title of this PR is a good option that follows our preferred style. When you apply this suggestion, make sure to squash/fixup your commits into one.